Increase scope of use

GPKyte opened the pull request:
Added new tables to track users and channels.
Added export feature to create Slack-ish export
Changed naming convention
Improved error-handling messages and comments

GPKyte committed:
Comments and Error Handling

Added helpful comments for when importing goes wrong, and added some
simple comments to the bot

GPKyte committed:
Track new channels and users

While keeping the same naming for compatibility, added two tables to
track the users and channels as they are updated.

GPKyte committed:
New tables and export feature

Added new tables to track users and channels.
Added export feature to create Slack-ish export
Changed naming convention

docmarionum1 commented on archivebot.py:
```diff
@@ -1,178 +1,211 @@
+            m['id'],
+            m['profile'].get('image_32', 'https://secure.gravatar.com/avatar/c3a07fba0c4787b0ef1d417838eae9c5.jpg?s=32&d=https%3A%2F%2Ffst.slack-edge.com%2F66f9%2Fimg%2Favatars%2Fava_0024-32.png')
+        ))
+    cursor.executemany("INSERT INTO users(name, id, avatar) VALUES(?,?,?)", args)
```
I get an error here due to the create table command having `text avatar` instead of `avatar text`.

GPKyte replied:
Forgot that I changed this on my branch but not on this pull, thanks for pointing it out. I pushed the updates.

GPKyte committed:
Readability and avatar fix

Update picture saved for avatar thumbnails to image_72
Fixed table error with avatar
Added status updates and line separators

docmarionum1 commented:
Thanks @GPKyte !  I made a couple of changes to `export.py` to make it python3 compatible.  Do you want to double check and make sure it still works for you?

GPKyte commented:
This looks good to me! Thanks @docmarionum1

> On May 29, 2017, at 12:02 PM, Jeremy Neiman <notifications@github.com> wrote:
> 
> Thanks @GPKyte ! I made a couple of changes to export.py to make it python3 compatible. Do you want to double check and make sure it still works for you?
> 
> ―
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub, or mute the thread.
>
